guides/common/modules/proc_migrating-project-to-el-9-by-using-cloning.adoc

```diff
@@ -0,0 +1,24 @@
+[id="migrating-{project-context}-to-el-9-by-using-cloning_{context}"]
+= Migrating {Project} to {EL-abbr}{nbsp}9 by using cloning
+
+You can clone your existing {ProjectServer} from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+ifdef::satellite[]
+Create a backup of the existing {ProjectServer}, which you then clone on the new {EL}{nbsp}9 system.
+[NOTE]
+====
+You cannot use cloning for {SmartProxyServer} backups.
+====
+endif::[]
+
+.Procedure
+. Perform a full backup of your {ProjectServer}.
+This is the source {EL}{nbsp}8 server that you are migrating.
+For more information, see {AdministeringDocURL}Performing_a_Full_Backup_admin[Performing a full backup of {ProjectServer}] in _{AdministeringDocTitle}_.
+. Deploy a system with {EL}{nbsp}9 and the same configuration as the source server.
+This is the target server.
+ifdef::satellite[]
+. Clone the server.
+Clone configures hostname for the target server.
+. Test the newly configured server before retiring the source server.
```

Comment:
There is no "test the new box" in the backup/restore version of this, so I wonder if that's really worth mentioning? (Plus, as a user, I might start asking: what exactly do I need to test, and we don't give an answer to that)